Playbooks: Multiple runs (#6148)
mattermost/docs: source/playbooks/work-with-playbooks.rst

5. Choose **Save**.

Create fewer channels
---------------------

> From Mattermost v7.7, you can choose to start each run in a new channel or re-use the same channel.

Reviewer: ```suggestion
From Mattermost v7.7, you can choose to start each run in a new channel or re-use an already existing channel.
```

not sure about the exact wording, but the concept of "same channel" seems a bit awkward because when I start reading the sentence, in my mind I'm not in the context of any channel. 

Additionally, the existing-channel / new-channel are the exact words that we use in the product. Not sure if there are the best, but I'd like to use similar expressions both product and docs.

Here are some scenarios why you might want to start each run in the same channel:

- Short, frequently-used processes benefit from being in the same channel - it keeps the process streamlined.
- Teams with multiple independent workflows, such as release teams, benefit from having them in one place.
- Cutting down on the number of new channels created makes it easier to find run channels again.
- The run name isn't linked to the the channel's name so you can tell multiple workflows apart.

When you're configuring your playbook:

- You can link it to an existing channel so that each run starts in that channel.
- You can choose that each time the playbook is run, it creates a new channel.

To access this setting, open the playbooks tab. Select the playbook you want to edit, then select the **Outline** tab. Select **Actions** in the left-hand menu and make your selection under the **When a run starts** heading.

> When you start a run, your selection is the default but can be changed for each run.

Reviewer: would be this a good place to introduce "move run to a different channel" feature?

Something like, "Additionally, it is also possible to move a started run to another channel if that is needed"

Author: Good idea, thanks! :)

> When you start a run, your selection is the default but can be changed for each run.

Author: ```suggestion
When you start a run, your selection is the default but can be changed for each run. Additionally, it's also possible to move a started run to another channel, so you're not locked into whichever option you select.
```
